azure-kusto-ingest/README.rst

```diff
@@ -4,10 +4,10 @@ Microsoft Azure Kusto Ingest Library for Python
 .. code-block:: python
 
     from azure.kusto.data import KustoConnectionStringBuilder
-    from azure.kusto.ingest import KustoIngestClient, IngestionProperties, FileDescriptor, BlobDescriptor, DataFormat
+    from azure.kusto.ingest import QueuedIngestClient, IngestionProperties, FileDescriptor, BlobDescriptor, DataFormat

```

Comment:
leftover

Reply:
Had to invalidate and clear IDE's cache in order for it to identify these, for some reason.